### PR TITLE
Set all external links to open in new browser windows

### PR DESCRIPTION
**Pull request by rclations**
Lots of external links in the plugin are already set to open in new windows using `target="_blank"`. 
This pull request tackles the remaining ones that don't currently open in new windows.

I reviewed every instance of `href`in the plugin and added `target="_blank"` where appropriate.

### CODE REVIEW

**Comment by dereksmart** [comment]
@rclations thank you!

You can omit the changes from csstidy, as it's an external library.  https://github.com/Cerdic/CSSTidy/blob/master/lang.inc.php

**Comment by eliorivero** [comment]
Thanks @rclations for the PR! It needs a rebase now. Can you please refresh it?

**Comment by rclations** [comment]
no problem! I think it should be all set now - if you need anything else, just give me a shout :)

**Comment by eliorivero** [comment]
Thank you!

File: modules/module-info.php [hunk 1]
```diff
@@ -855,7 +855,7 @@ function jetpack_custom_content_types_more_info() { ?>
 
 			<p><?php esc_html_e( 'Once activated, you can selectively activate the content types you need at Settings &rarr; Writing &rarr; Your Custom Content Types.'); ?></p>
 
-            <p><?php echo wp_kses( sprintf( __( 'Full details can be found on the <a href="%s" title="Custom Content Types support page" target="_blank">Custom Content Types support page</a>.', 'jetpack' ), 'https://jetpack.com/support/custom-content-types/' ), array( 'a' => array( 'href' => true, 'title' => true, 'target' => true ) ) ); ?></p>
+            <p><?php echo wp_kses( sprintf( __( 'Full details can be found on the <a href="%s" "title="Custom Content Types support page" target="_blank">Custom Content Types support page</a>.', 'jetpack' ), 'https://jetpack.com/support/custom-content-types/' ), array( 'a' => array( 'href' => true, 'title' => true, 'target' => true ) ) ); ?></p>
```
Comment (eliorivero) [inline modules/module-info.php]: @rclations ouch! that double quote would break something :) can you please fix?

Comment (rclations) [inline modules/module-info.php]: ouch indeed! my apologies on that - it should be fixed now.

**Comment by eliorivero** [comment]
Thanks! this is good to go now :+1:

**Comment by samhotchkiss** [comment]
Thanks, @rclations !

**Pull request merged**